cf-ops-automation-broker-core/src/test/java/com/orange/oss/cloudfoundry/broker/opsautomation/ondemandbroker/pipeline/tools/TreeTest.java

```diff
@@ -1,12 +1,38 @@
 package com.orange.oss.cloudfoundry.broker.opsautomation.ondemandbroker.pipeline.tools;
 
+import com.orange.oss.cloudfoundry.broker.opsautomation.ondemandbroker.pipeline.StructureGeneratorHelper;
 import org.junit.Ignore;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.hamcrest.MatcherAssert.assertThat;
 
 public class TreeTest {
 
-    @Test@Ignore
-    public void test() {
-        //(new Tree()).print("/home/losapio/GIT/Coab/paas-templates/coab-depls/mongodb");
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+
+    @Test
+    public void test() throws IOException {
+        //Given a root path and path elements to create
+        Path rootPath = this.temporaryFolder.getRoot().toPath();
```

Comment:
@JCL38-ORANGE 
What about using files in src/test/resources/ tree such as a reference data set to execute tree against (e.g. copy of sample-deployment-model which can include both symlinks, regular files and directories), in order to avoid the java code to set up these files ?